shader/antialiasing/fxaa.frag

```diff
@@ -0,0 +1,87 @@
+#version 450
+
+#extension GL_GOOGLE_include_directive : enable
+
+#define FXAA_GLSL_130 1
+
+#if FXAA_QUALITY_SETTING == 0
+	#define FXAA_QUALITY_PRESET 10
+	#define FXAA_PC_CONSOLE 1
+#elif FXAA_QUALITY_SETTING == 1
+	#define FXAA_QUALITY_PRESET 10
+	#define FXAA_PC 1
+#elif FXAA_QUALITY_SETTING == 2
+	#define FXAA_QUALITY_PRESET 15
+	#define FXAA_PC 1
+#elif FXAA_QUALITY_SETTING == 3
+	#define FXAA_QUALITY_PRESET 29
+	#define FXAA_PC 1
+#elif FXAA_QUALITY_SETTING == 4
+	#define FXAA_QUALITY_PRESET 39
+	#define FXAA_PC 1
+#endif
+
+#include "Fxaa3_11.h"
+
+layout(push_constant, std140) uniform PushConstantsFxaa {
+  float fxaaInverseSharpnessCoeff;
+  float fxaaQualitySubpix;
+  float fxaaQualityEdgeThreshold;
+  float fxaaQualityEdgeThresholdMin;
+  float fxaaConsoleEdgeSharpness;
+  float fxaaConsoleEdgeThreshold;
+  float fxaaConsoleEdgeThresholdMin;
+  } pushConstantsFxaa;
+
+layout(binding  = 0) uniform sampler2D aliasedInput;
+
+layout(location = 0) in  vec2 uv;
```

Comment:
Should be a `gl_FragCoord.xy / screenSize`. Asking, as there is an intent to get rid of `-DHAS_UV` option in post-process and make it more compatible with compute